src/main/java/ch/uzh/ifi/hase/soprafs24/entity/Session.java

```diff
@@ -18,10 +18,15 @@ public class Session implements Serializable {
     @Getter
     @Setter
     @Column
-    private String turnPlayer;
+    private Integer turnPlayer;
```

Comment:
Why is turnPlayer an Integer now? Shouldn't it hold a playerId?

Reply:
since we have an orderIndex in player i figured we could just say what orderIndex currently has his turn

Reply:
Got it, yes we can do it like this im on board